04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+\end{definition}
+
+\begin{definition} \label{def:Maximising Social Welfare}
+\textbf{Maximising Social Welfare} is a strategy which attempts to maximise the total amount of welfare (resources) the islands have.
+\end{definition}
+
+
+\subsection{Foraging Background}
+
+Foraging is a fundamental requirement that is needed as it is a method of obtaining more resources from the starting amount. Foraging is based on the concept of inputting resources to get a return.
+
+\subsubsection{Underlying Concept}
+
+The primary objective of foraging in this game is to introduce a dilemma to the agents in order to investigate their behaviours and whether they work together despite different personal goals. Including two methods of foraging promotes decision making among the islands, allowing for strategic interactions to maximise personal benefits while still reaching the overarching goal of survival. 
+
+The inspiration for our concept of foraging comes from the stag hunt game. Thus, it introduces elements from it. Specifically, this approach results in a game with \textbf{no dominant strategy} and in which \textbf{an agent's choice can impact another agent's}.
+
+The process of stag hunt is that each agent selects a method of hunting without the information of the other agents. The social welfare (total payoff) is significantly reduced if their choices differ. The selection of stag suggests that the agent is going for higher \textbf{payoffs} whilst the selection of hare indicates that the agent is \textbf{risk aversion} as both agents are required to stag hunt to get any personal welfare (see Definition~\ref{def:Welfare}).
+
+\subsubsection{Modifications to the Original concepts} 
+A few changes have to be made to make this dilemma viable for our multi-agent system, in addition to adding realism. 
+
+\begin{enumerate}
+    \item The dependency was changed to be based on the input resources rather than the number of islands participating. This added a more realistic element to the dilemma as in reality a hunt’s return would be based on the amount of resources (different possible resources: people, food, water and materials) entered rather than the number of islands participating.
+    \item Instead of a specified return, probabilistic return was implemented which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
+\end{enumerate}
+
+Furthermore, the dynamics of our agents and allowing for a more complex decision-making interaction between islands and the environment have been explored. It is worth mentioning that some of these design ideas were implemented in the coursework. These changes have been made to make foraging more volatile and unpredictable, adding to the complexity of the system, allowing the teams to further evaluate the agents’ performance on observing and to use learned knowledge to take action.
```

Comment:
reallly long sentence at the end here - easy fix might be to cut it into two by making the last part: This allows the teams...